FetchRequests/Sources/Controller/FetchedResultsControllerProtocol.swift

```diff
@@ -7,12 +7,19 @@
 //
 
 import Foundation
+import Combine
 
-internal protocol InternalFetchResultsControllerProtocol: AnyObject, FetchedResultsControllerProtocol {
+internal protocol InternalFetchResultsControllerProtocol: FetchedResultsControllerProtocol {
     func manuallyInsert(objects: [FetchedObject], emitChanges: Bool)
 }
 
-public protocol FetchedResultsControllerProtocol {
+public protocol DoublyObservableObject: ObservableObject {
```

Comment:
Is `Doubly` a conventional name?

Reply:
I'm thinking doubly linked list

Reply:
I don't _really_ expect anyone to use the protocol, I just did it for organizational purposes